auto-complete.el

```diff
@@ -1671,7 +1671,8 @@ that have been made before in this function.  When `buffer-undo-list' is
       (when (and ac-triggered
                  (or ac-auto-start
                      ac-completing)
-                 (not isearch-mode))
+                 (not isearch-mode)
+                 (looking-back "\\>"))
```

Comment:
I don't think this is necessary, you could just simply move the cursor to the end of the word?